docs/users/user-guide.md

```diff
@@ -2,15 +2,59 @@
 
 ## Finding projects
 
+Published projects can be browsed at [https://biosimulations.org/projects](https://biosimulations.org/projects). Each card presents a project, with a thumbnail, and title. Mousing over the thumbnail shows additional details about the project. You can customize the attributes you want to view. 
+
+![browse-projects](./images/browse.png)
+### Selecting attributes
+
+![select-attributes](./images/select.png){align=right}
+Clicking on the search icon in the top-right corner of the page opens menu with an attributes sub-menu. From here, you can select the attributes you want to view. Selecting a field will add the attributes to the details presented in the project card when you mouse over the thumbnail.
+
+ <!-- The new lines must be followed by two spaces-->
+&NewLine;  
+&NewLine;  
+&NewLine;  
+&NewLine;  
+&NewLine;  
+&NewLine;  
+
+### Searching for projects
+![search-projects](./images/search.png){align=right}
+Clicking on the the search icon at the top right of the page opens a search box. A search term, such as 'metabolism' can be entered in the search box. By default, the search term is searched against each [attribute](#selecting-attributes) of the project. Optionally, you can restrict the search to a specific attribute. For example, if you want to search for projects that have the value 'Eschiria coli' in the `taxa`  attribute, you can enter 'taxa:eschiria-coli' in the search box. The general format for queries is `attribute:value-with-dashes`. The name of the attribute, followed by a colon, followed by the search term, replacing all spaces with '-'.
```

Comment:
"Eschiria" is spelled "Escherichia"

The dashes should be used in the attribute names. A list of the attributes is in the FAQs page.

I'm not sure that this the lunr search syntax for multiword phrases. There doesn't appear to be [documentation](https://lunrjs.com/guides/searching.html) for this.